crates/credential/cargo-credential/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "cargo-credential"
+version = "0.1.0"
+authors = ["The Rust Project Developers"]
+edition = "2018"
+license = "MIT OR Apache-2.0"
```

Comment:
This is intended for publication, right? If so want to fill in some metadata here?

Reply:
(same for the other crates)